src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -76,9 +76,15 @@ def legacy_dir(self):
 
     @property
     def legacy_config_path(self):
-        m = md5()
-        m.update(self.base_path)
-        meta_config = "{0}.meta".format(m.digest().hex())
+        try:
+            m = hashlib.md5(self.base_path)
+        except ValueError:
+            try:
+                m = hashlib.md5(self.base_path, usedforsecurity=False) # type: ignore
```

Comment:
@ajarr @batrick Ignored mypy error and switched the order. Assuming non fips systems are comparatively more. PTAL

Reply:
got it.